hmt_escrow/job.py

```diff
@@ -459,19 +447,24 @@ def bulk_payout(self,
 
         Args:
             payouts (List[Tuple[str, int]]): a list of tuples with ethereum addresses and amounts.
-            results (Dict): the final answer results stored by the Reputation Oracle.
+            results (Dict): the repO answer results stored by the Reputation Oracle.
             pub_key (bytes): the public key of the Reputation Oracle.
+            gas (int): Optional, gas limit
+            storeOnchain (bool): Store data onchain. Saves 10k gas
 
         Returns:
             bool: returns True if paying to ethereum addresses and oracles succeeds.
 
         """
-        (hash_, url) = upload(results, pub_key)
+        (hash_, url) = upload(results, pub_key, ipnsKeypairName='reputation-oracle-results-' + self.job_contract.address)
```

Comment:
often useful to do a f type string as its a bit cleaner: `f'reputation-oracle-results-{self.job_contract.address}'`

Reply:
done